src/c4/ifpstream.cc

```diff
@@ -0,0 +1,69 @@
+//----------------------------------*-C++-*-----------------------------------//
```

Comment:
Please provide a better PR title.  Only the titles go into the ChangeLog and "Parallel read" isn't very descriptive.  Thanks!

Reply:
Fixed.

Reply:
Fixed.